src/blk/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ if(WITH_SPDK)
 endif()
 
 if(HAVE_LIBZBC)
-  target_link_libraries(blk ${ZBC_LIBRARIES})
+  target_link_libraries(blk PUBLIC ${ZBC_LIBRARIES})
```

Comment:
@agayev can we link against libzbc with PRIVATE? as i think the header files are included in the `.cc` file . and we don't expect `libblk` to expose symbols of libzbc, right?

Reply:
could you split this change into two? one for renaming zns to zoned, another for adding the libzbc linkage with keyword.

Reply:
@tchaikov sure, I've created two new PRs for this:

https://github.com/ceph/ceph/pull/35479
https://github.com/ceph/ceph/pull/35480

Reply:
sorry! i meant to say two commits. as two PRs would two merge commits. so i'd prefer have two commits instead of two PRs. we have multiple PRs for reduce the overhead of reviewer and the risk of tests.

Reply:
No worries.  I will turn this into two commits and delete the PRs.

Reply:
@tchaikov Does this look okay now?  Please let me know if it doesn't.  It will take some trials and errors to catch up with Ceph project conventions.

Reply:
thanks, LGTM!